services/app-api/serverless.yml

```diff
@@ -23,23 +23,14 @@ custom:
   applicationEndpoint: ${env:APPLICATION_ENDPOINT, cf:ui-${sls:stage}.CloudFrontEndpointUrl}
   appApiGatewayId: ${env:CF_CONFIG_IGNORED_LOCALLY, cf:infra-api-${sls:stage}.ApiGatewayRestApiId}
   appApiGatewayRootResourceId: ${env:CF_CONFIG_IGNORED_LOCALLY, cf:infra-api-${sls:stage}.AppApiGatewayRootResourceId}
-  emailSource: ${env:SES_SOURCE_EMAIL_ADDRESS}
   emailerMode: ${env:EMAILER_MODE}
-  parameterStoreMode: ${env:PARAMETER_STORE_MODE, ssm:/configuration/parameterStoreMode}
```

Comment:
`parameterStoreMode` is being removed, are we getting it form somewhere else now? This was added for my work with configuring parameter store mode in `app-api/src/handlers/apollo_gql.ts`.

Reply:
Good catch! I think this got blown away when pulling in `main` and fixing up the merge conflict. Adding it back in.